taskt/Core/Automation/Commands/Variable Commands/AddListItemCommand.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Microsoft.Office.Interop.Outlook;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Windows.Forms;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
+
+namespace taskt.Core.Automation.Commands
+{
+    [Serializable]
+    [Group("Variable Commands")]
+    [Description("This command adds an item to an existing list variable.")]
+    public class AddListItemCommand : ScriptCommand
+    {
+        [XmlAttribute]
+        [PropertyDescription("List Item")]
+        [InputSpecification("Enter the list item for the variable.")]
+        [SampleUsage("Hello || {vText}")]
+        [Remarks("List item can only be a String, DataTable, MailItem or IWebElement.")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        public string v_ListItem { get; set; }
+
+        [XmlAttribute]
+        [PropertyDescription("Output List Variable")]
+        [InputSpecification("Select or provide a variable from the variable list.")]
+        [SampleUsage("vUserVariable")]
+        [Remarks("If you have enabled the setting **Create Missing Variables at Runtime** then you are not required" +
+                  " to pre-define your variables; however, it is highly recommended.")]
+        public string v_OutputUserVariableName { get; set; }
```

Comment:
I looked at the code and this looks like an input, not an output. You provide the user with the list variable and the item you want to add to it and it adds it.

Reply:
You can use GetListItemCommand as a model for consistency.

Reply:
Updated